theme/colorsDark.ts

```diff
@@ -49,6 +49,7 @@ export const colorsDark: IColors = {
     secondary: lightPalette.light60,
     tertiary: lightPalette.light30,
     inactive: lightPalette.light15,
+    caution: darkPalette.red,
```

Comment:
Should this be lightPalette instead?

Reply:
Probably my error but yes we should use lightPalette, even if it's the same color dark/light for now